lib/oprt.gd

```diff
@@ -1334,18 +1334,27 @@ OrbitishFO( "Orbit", OrbitishReq, IsCollsElms, false, false );
 #############################################################################
 ##
 #O  Orbits( <G>, <seeds>[, <gens>, <acts>][, <act>] )
+#A  Orbits( <G> )
 #A  Orbits( <xset> )
 ##
 ##  <#GAPDoc Label="Orbits">
 ##  <ManSection>
 ##  <Oper Name="Orbits" Arg='G, seeds[, gens, acts][, act]' Label="operation"/>
+##  <Attr Name="Orbits" Arg='G'
+##   Label="for a permutation group"/>
 ##  <Attr Name="Orbits" Arg='xset' Label="attribute"/>
 ##
 ##  <Description>
 ##  returns a duplicate-free list of the orbits of the elements in
 ##  <A>seeds</A> under the action <A>act</A> of <A>G</A> or under
 ##  <Ref Func="OnPoints"/> if no action function is given.
 ##  <P/>
+##  For a permutation group <A>G</A>, one may also invoke this as
```

Comment:
Maybe you could use this shorter version:
_For a permutation group <A>G</A>, <C>Orbits(<A>G</A>)</C> computes the orbits on the moved points of <A>G</A>._

Reply:
This is the outcome of another conversation in [PR 3363](https://github.com/gap-system/gap/pull/3363#discussion_r267409263). I think both of the two options are fine. So I leave it as it is for now, unless you or someone else has strong objections.

Reply:
Ok, I wasn;t aware of the discussion in #3363 .